paper-slider.html

```diff
@@ -90,6 +90,10 @@
       :host(:focus) {
         outline: none;
       }
+        
+      :host-context([dir="rtl"]) {
```

Comment:
to be fair, there's a SLEW of stuff that's going away in Shadow DOM V1.  `<sarcasm>`do you want us to remove the `<content>` element while we're here?`</sarcasm>`

sure, we could do something like this:

```html
<style>
  :host([rtl_]) {
    /* rtl-y stuff */
  }
</style>
<script>
Polymer({
  is: 'paper-slider',

  properties: {
    rtl_: {
      type: Boolean,
      reflectToAttribute: true,
    },
  },

  attached: function() {
    // Can't do this until we're in the document.
    this.rtl_ = getComputedStyle(this).direction == 'rtl';
  },
});
</script>
```

but it's a bunch more code (and I'd argue it's not perfect because it's done once on `attached`, just like `:host-context()` with multiple `[dir]` ancestors)